src/errors.rs

```diff
@@ -18,4 +18,7 @@ pub enum KaboodleError {
 
     #[error("Unable to stop: {0}")]
     StoppingFailed(String),
+
+    #[error("Functionality unavailable because the mesh is not currently running")]
+    NotRunning,
```

Comment:
Thanks! Humourously, yesterday I realized that I don't really need it 😮‍💨 In the original implementation of this where `KaboodleInner` sent peer arrivals over a channel to `Kaboodle`, it was a requirement (since there's only a KaboodleInner instance while the mesh is actually running). Now that `Kaboodle` is observing the known_peers map (which always exists), I don't need it. 🙃